src/Scripts/Python/PythonModule.cs

```diff
@@ -80,7 +83,7 @@ public override void FireEvent(ScriptEvent @event, Program program)
             catch (Exception ex)
             {
                 var scriptError = CreateError(
-                    Filename,
+                    Uri.ExtractString(),
```

Comment:
I'm using `ImageLocation.FileSystemPath` now, since currently the code doesn't support running scripts inside of archives.
Reko would have to extract/unpack them first to a temporary file, before the Python engine could execute them.